sw/airborne/firmwares/rotorcraft/autopilot_static.c

```diff
@@ -317,7 +311,7 @@ void autopilot_static_set_mode(uint8_t new_autopilot_mode)
 
 void autopilot_static_set_motors_on(bool motors_on)
 {
-  if (autopilot.mode != AP_MODE_KILL && ap_ahrs_is_aligned() && motors_on) {
+  if (autopilot.mode != AP_MODE_KILL && motors_on) {
```

Comment:
I'm still not sure about the implication here if "check" module is not used and someone is trying to start motors before ahrs is aligned.

Reply:
Yes that is indeed removed now by default. My idea was to automatically load the preflight_check module, such that this is re-added to all platforms by default. Loading the preflight_check module by default might also prevent a lot of failures/crashes of inexperienced users when we add more checks.
Is it possible to unload an automatically added module?